docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+chosen, you can check as follows:
+
+ #. Log in to the *Application Server* via the console using the adminstration username
+    and password.
+ #. Check to see if a file ``/var/lib/services/tor/ssh/hostname`` exists. If it
+    does, your instance is set up to use SSH over Tor and you should configure 
+    temporary SSH access using :ref:`these instructions <rebuild_ssh_over_tor>`.
+    If not, your instance is set up to use SSH over LAN, and you should follow
+    :ref:`these instructions instead <rebuild_ssh_over_lan>`.
+
+.. _rebuild_ssh_over_tor:
+
+Configuring access for an SSH-over-Tor instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Direct SSH access is disabled when the SSH-over-Tor option is selected during
+installation. To temporarily re-enable it, you'll need to update ``iptables`` rules
+and change the sshd daemon's configuration.
+
+First, log on to the *Application Server* via the console, and run the following
+commands, substituting the *Admin Workstation's* static IP for ``<admin_static_ip>``:
+
+.. code:: sh
+
+  sudo iptables -I INPUT -p tcp -dport 22 -s <admin_static_ip> \
+    -m state --state NEW,ESTABLISHED -j ACCEPT
+  sudo iptables -I OUTPUT -p tcp -sport 22 -m state --state ESTABLISHED -j ACCEPT 
+
+Next, edit the file ``/etc/ssh/sshd_config``, changing the line:
+
+.. code-block:: none
+
+  ListenAddress 127.0.0.1:22                                    
+
+to:
+ 
+.. code-block:: none
+
+  ListenAddress 0.0.0.0:22                                   
+
+and deleting the line: 
+
+.. code-block:: none
+
+  PasswordAuthentication no                                         
+
+Then, restart ``sshd`` using the command ``sudo service sshd restart``.
+
+Finally, check the file ``/etc/networking/interfaces``, and note down the network
+settings for the default Ethernet interface. You'll need them in the next step.
+
+Repeat the process above for the *Monitor Server*, making sure to note down its
+network settings as well.
+
+.. _rebuild_ssh_over_lan:
+
+Configuring access for an SSH-over-LAN instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, log on to the *Application Server* via the console and edit the file 
+``/etc/ssh/sshd_config``, deleting the line:
+
+.. code-block:: none
+
+  PasswordAuthentication no                                         
+
+Then, restart ``sshd`` using the command ``sudo service sshd restart``.
+
+Finally, check the file ``/etc/networking/interfaces``, and note down the network
+settings for the default Ethernet interface. You'll need them in the next step.
+
+Repeat the process above for the *Monitor Server*, making sure to note down its
+network settings as well.
+
+Enabling access from the new *Admin Workstation*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the *Admin Workstation*, open a terminal and copy the *Admin Workstation's* 
+SSH public key to the servers, substituting the appropriate values for the username
```

Comment:
"the appropriate values" feels a bit handwavy -- I suggest we explicitly direct the user to the "address" definition in `/etc/network/interfaces`.

Reply:
removed handwaviness.